src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -259,6 +259,8 @@ protected void setup()
 
         CommitLog.instance.start();
 
+        installCryptoProvider();
```

Comment:
Why not to do this installation logic in `CassandraDaemon.activate()` method? There is `applyConfig()` at the very beginning. Then it will eventually propagate to `DatabaseDescriptor.applyAll()`.